cli.js

```diff
@@ -28,6 +29,12 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+const pkg = readPkgUp.sync().pkg;
+if (!pkg) {
+	console.error('\nNo package.json found, are you in the correct project?');
```

Comment:
That's because we don't want the error/message being printed directly after the execution list.